### DISCUSSION TITLE
Mail command to send temporary mail to all players instead of just one person

### DISCUSSION THREAD

**Discussion by TheIntolerant** [EssentialsX/Essentials#4558] [Ideas and Feature Suggestions] [answered]
### Suggestion: Add a way to send temporary mails to all people.
In the latest stable update of EssentialX there was the addition of
``/mail sendtemp <player> <expire time> <message>``
I was sad to realise you couldn’t  send temp mails to all players and think this is something that should be added.

### Why should this be added?
The addition of such commands would be extremely helpful in the case of server wide events etc, in which case it wouldn’t be ideal for mails to still exist that was about an Event that finished a while ago.

Thank you for taking the time to read this and I hope you consider this suggestion.

**Comment by JRoy** [comment, marked as answer]
Will be added in https://github.com/EssentialsX/Essentials/pull/4561
